Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -885,6 +884,262 @@ end
     break
 end)
 
+# This is the interpreted analogue to WasmBinding.cpp:wasmToWasm
+op(wasm_to_wasm_wrapper_entry, macro()
+    loadq Callee-0x10[sp], ws1
+    bilt ws1, 0, .funcindex
+
+    leap Wasm::Callee::m_indexOrName[ws1], ws1
+    loadq Wasm::IndexOrName::m_indexName[ws1], ws1
+if JSVALUE64
+    const indexTag = (constexpr Wasm::IndexOrName::indexTag)
+else
+    const indexTag = 0
+end
+    andq ~indexTag, ws1
+    jmp .done
+
+.funcindex:
+    noti ws1
```

Comment:
I wonder if we can just get function index from boxed callee. You can unbox and load index from that.